fontbe/src/features.rs

```diff
@@ -106,7 +141,32 @@ impl Work<Context, Error> for FeatureWork {
             .iter()
             .map(|n| Into::<FeaRsGlyphName>::into(n.as_str()))
             .collect();
-        let font = self.compile(fea_file, glyph_map)?;
+
+        let font = match &*features {
+            Features::File(fea_file) => self.compile_file(fea_file, glyph_map)?,
+            Features::Memory(fea_content) => {
+                let result = self.compile_memory(fea_content, glyph_map);
+                if result.is_err() || context.emit_ir {
```

Comment:
not sure if emit_ir flag should also apply to emitting debug files, they aren't really the same thing and one may want one without the other? maybe you could have a separate --emit-debug or something

Reply:
Oh good call. Will add.